dvc/repo/status.py

```diff
@@ -74,10 +73,8 @@ def _cloud_status(
     )["local"]
 
     ret = {}
-    status_info = self.cloud.status(
-        used, jobs, remote=remote, show_checksums=show_checksums
-    )
-    for md5, info in status_info.items():
+    status_info = self.cloud.status(used, jobs, remote=remote)
+    for _, info in status_info.items():
```

Comment:
I guess we could use `status_info.values()` instead, but at that point we might refactor this so it is efficiently using filter() and map(), but that would be a separate task. Having this as-is is fine for now.

Reply:
I have made the changes. @efiop I am curious what is the use of files in  directory `command` and `status` in this project.

Also I couldn't find any command with data_sync.py in folder `command`. Why is it like that?

Reply:
@kurianbenoy

> I have made the changes.

Please see another comment down below.

>  I am curious what is the use of files in directory command and status in this project.
> Also I couldn't find any command with data_sync.py in folder command. Why is it like that?

just legacy files. push/pull/fetch/status were called "data sync" a while ago.

Reply:
Ok thanks for clarrifying